e3/fingerprint.py

```diff
@@ -225,12 +225,16 @@ def load_from_file(cls, filename):
         Return None in the following situations:
             - The file does not contain a fingerprint we recognize;
             - The fingerprint has a version number we do not support.
+            - If the fingerprint file does not exist
 
         :param filename: The name of the file where to load the fingerprint
             from.
         :type filename: str
         :rtype: Fingerprint
         """
+        if not os.path.isfile(filename):
```

Comment:
This change means we can simplify the "load_previous_fingerprint" method in class ElectrolytWalk:

``        if os.path.exists(filename):
            return Fingerprint.load_from_file(filename)
        else:
            return None
``